presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMaterializedViewLogicalPlanner.java

```diff
@@ -1077,6 +1089,551 @@ public void testBaseToViewConversionWithGroupBy()
+                                                                                    ImmutableList.of(),
+                                                                                    ImmutableMap.of(),
+                                                                                    Optional.empty(),
+                                                                                    PARTIAL,
+                                                                                    node(
+                                                                                            ProjectNode.class, constrainedTableScan(
+                                                                                                    table,
+                                                                                                    ImmutableMap.of("ds", multipleValues(createVarcharType(10), utf8Slices("2021-07-11", "2021-07-12"))),
+                                                                                                    ImmutableMap.of("orderkey", "orderkey", "ds", "ds")))))))))))));
+
+            //expect a query on the base table to behave as if the MV doesn't exist
+            PlanMatchPattern expectedPatternWithoutCountOptimization = anyTree(
+                    aggregation(
+                            singleGroupingSet("orderkey"),
+                            ImmutableMap.of(Optional.empty(), functionCall("count", false, ImmutableList.of(anySymbol()))),
+                            ImmutableList.of(),
+                            ImmutableMap.of(),
+                            Optional.empty(),
+                            FINAL,
+                            node(
+                                    ExchangeNode.class,
+                                    anyTree(
+                                            aggregation(
+                                                    singleGroupingSet("orderkey"),
+                                                    ImmutableMap.of(Optional.empty(), functionCall("count", ImmutableList.of())),
+                                                    ImmutableList.of(),
+                                                    ImmutableMap.of(),
+                                                    Optional.empty(),
+                                                    PARTIAL,
+                                                    constrainedTableScan(
+                                                            table,
+                                                            ImmutableMap.of("ds", multipleValues(createVarcharType(10), utf8Slices("2021-07-11", "2021-07-12"))),
+                                                            ImmutableMap.of("orderkey", "orderkey")))))));
+
+            assertPlan(getSession(), viewQuery, expectedPatternWithCountOptimization);
```

Comment:
Looks like names expectedPatternWithCountOptimization and expectedPatternWithoutCountOptimization exchanged?